src/Components/CreateImageWizard/steps/Registration/components/RegisterLaterAlert.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { Alert, Button, Content } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { INSIGHTS_URL, RHC_URL } from '../../../../../constants';
+
+const RegisterLaterAlert = () => {
+  return (
+    <Alert
+      variant="info"
+      title={
+        <>
+          You should register your image now if you may want to connect to Red
```

Comment:
Is this the copy we were given?

```
You should register your image now if you would like to connect to Red Hat insights...
```

Sounds slightly better to me.

Reply:
Yup, this is taken directly from the mocks: https://www.figma.com/design/9t3owYlJi4M23fUXWpgXCE/HMS-Image-Builder?node-id=3230-48342&t=r5LcLuq1Se2acEyk-0

Reply:
@kelseamann what do you think?